tests/php/ORM/DataListTest.php

```diff
@@ -1967,24 +1967,21 @@ public function testSortScalarValues(mixed $emtpyValue, string $type): void
         $this->assertSame(['Subteam 1'], Team::get()->limit(1)->column('Title'));
         $list = Team::get()->sort('Title DESC');
         $this->assertSame(['Team 3'], $list->limit(1)->column('Title'));
-        if ($type !== 'wipes-existing') {
-            $this->expectException(InvalidArgumentException::class);
-        }
+        $this->expectException(InvalidArgumentException::class);
         if ($type === 'invalid-scalar') {
             $this->expectExceptionMessage('sort() arguments must either be a string, an array, or null');
         }
         if ($type === 'empty-scalar') {
             $this->expectExceptionMessage('Invalid sort parameter');
         }
-        // $type === 'wipes-existing' is valid
+
         $list = $list->sort($emtpyValue);
         $this->assertSame(['Subteam 1'], $list->limit(1)->column('Title'));
     }
 
     public function provideSortScalarValues(): array
     {
         return [
-            [null, 'wipes-existing'],
```

Comment:
This scenario was testing that calling `sort` with null unsets the order by clause. I've replaced it with a sightly different test below.